Command+E - Error

<!-- NOTE! The information you specify will be publicly accessible. -->
### Expected behaviour

### Actual behaviour

### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2023-10-24T14:31:10.014593</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>b364f346962cf24b6845a52c0c97beb5deb7a8f1</td></tr>
<tr><td>Defold engine sha</td><td>a90c50928623cf23b3687a7eec05972d11427202</td></tr>
<tr><td>Defold version</td><td>1.6.1</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"3cb6e896ec9a4462b266ff1bb8002741"'>3cb6e896ec9a4462b266ff1bb8002741</a></td></tr>
<tr><td>GPU</td><td>Apple M1 Pro</td></tr>
<tr><td>GPU Driver</td><td>2.1 Metal - 88</td></tr>
<tr><td>Java version</td><td>17.0.5+8</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>14.1</td></tr>
<table>

```
editor.scene_visibility$show_outline_name_paths_BANG_.invokeStatic.class java.lang.AssertionError: Assert failed: (set? (not-empty outline-name-paths))
    at editor.scene_visibility$show_outline_name_paths_BANG_.invokeStatic(scene_visibility.clj:204)
    at editor.scene_visibility$fn$fnk55645_positional__55648.invoke(scene_visibility.clj:249)
    at editor.scene_visibility$fn$fnk55645__55660$fn.invoke(scene_visibility.clj:243)
    at editor.scene_visibility$fn$fnk55645__55660.invoke(scene_visibility.clj:243)
    at clojure.lang.AFn.applyToHelper(AFn.java:154)
    at clojure.lang.AFn.applyTo(AFn.java:144)
    at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
    at clojure.lang.RestFn.invoke(RestFn.java:408)
    at editor.handler$invoke_fnk.invokeStatic(handler.clj:256)
    at editor.handler$run.invokeStatic(handler.clj:289)
    at editor.ui$execute_handler_ctx.invokeStatic(ui.clj:1192)
    at editor.ui$execute_command.invokeStatic(ui.clj:1200)
    at editor.ui$invoke_handler.invokeStatic(ui.clj:1231)
    at editor.ui$invoke_handler.invoke(ui.clj:1225)
    at editor.ui$invoke_handler.invokeStatic(ui.clj:1227)
    at editor.keymap$execute_command$fn.invoke(keymap.clj:612)
    at clojure.lang.AFn.run(AFn.java:22)
    at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:456)
    at java.security.AccessController.doPrivileged(AccessController.java:399)
    at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:455)
    at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
```

--- a/editor/src/clj/editor/debug_view.clj
+++ b/editor/src/clj/editor/debug_view.clj
@@ -47,7 +47,7 @@
 (def ^:private break-label "Break")
 (def ^:private continue-label "Continue")
 (def ^:private detach-debugger-label "Detach Debugger")
-(def ^:private start-debugger-label "Start / Attach")
+(def ^:private start-debugger-label "Start/Attach")
 (def ^:private step-into-label "Step Into")
 (def ^:private step-out-label "Step Out")
 (def ^:private step-over-label "Step Over")

--- a/editor/src/clj/editor/scene_visibility.clj
+++ b/editor/src/clj/editor/scene_visibility.clj
@@ -241,8 +241,11 @@
   (run [scene-visibility] (hide-outline-name-paths! scene-visibility (g/node-value scene-visibility :unselected-hideable-outline-name-paths))))
 
 (handler/defhandler :hide-toggle-selected :workbench
-  (active? [scene-visibility evaluation-context] true)
-  (enabled? [scene-visibility evaluation-context] true)
+  (active? [scene-visibility evaluation-context]
+           (g/node-value scene-visibility :active-scene-resource-node evaluation-context))
+  (enabled? [scene-visibility evaluation-context]
+            (or (g/node-value scene-visibility :selected-hideable-outline-name-paths evaluation-context)
+                (g/node-value scene-visibility :selected-showable-outline-name-paths evaluation-context)))
   (run [scene-visibility]
        (g/with-auto-evaluation-context evaluation-context
          (let [should-hide (g/node-value scene-visibility :selected-hideable-outline-name-paths evaluation-context)]

